src/YieldDistributor.sol

```diff
@@ -20,7 +21,7 @@ import {IYieldDistributor} from "src/interfaces/IYieldDistributor.sol";
  * @custom:coauthor kassandra.eth
  * @custom:coauthor theblockchainsocialist.eth
  */
-contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
+contract YieldDistributor is IYieldDistributor, OwnableUpgradeable, VotingMultipliers {
```

Comment:
TBH I'm a little dubious of the upgrade safety here. It would probably be best to implement VotingMultipliers to use [ERC7201](https://eips.ethereum.org/EIPS/eip-7201) storage layouts (like they do with [OwnableUpgradeable](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/723f8cab09cdae1aca9ec9cc1cfa040c2d4b06c1/contracts/access/OwnableUpgradeable.sol#L27)) to avoid storage layout collisions. It seems complicated but it's actually a pretty trivial one time calculation and some boilerplate code for accessing storage variables.

Reply:
I was thinking we should just deploy a new proxy for this version, and use the setter on the bread contract, as this would also help us resolve #96

Reply:
Is there a foreseeable problem deploying as a new proxy, thus changing the address? if there is not, i would prefer we deploy fresh, therefore do not need to worry about upgrade-safety, and not over-complicated things. but again, this is assuming there is no major problem with changing the address.

Reply:
Deploying a new contract is definitely the move in this instance but I mean in future instances where we maybe want to make some changes to the `VotingMultipliers.sol` contract. Namespacing the storage layouts ensure `YieldDistributor` and `VotingMultipliers` are using their own storage locations that won't clash, so if we want to add a variable to one or the other, we can do so safely. I can make a quick branch so show what it would look like.

Reply:
I also think there is a strong case to be made for `VotingMultipliers` to be it's own contract.

Reply:
What are the pros/cons here? 

I believe we should redeploy fresh to avoid upgrade safety issues and address #96 at some point

Reply:
#110 is also opened but I would really like to merge this and then maybe hand it off to someone else at a later point